server/src/main/java/org/opensearch/node/StreamManagerWrapper.java

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
I guess we do not need this class in this PR

Reply:
yes